java/client/src/test/java/glide/api/models/TransactionTests.java

```diff
@@ -818,15 +818,12 @@ InfScoreBound.NEGATIVE_INFINITY, new ScoreBoundary(3, false), new Limit(1, 2)),
         results.add(Pair.of(XGroupCreate, buildArgs("key", "group", "id")));
 
         transaction.xgroupCreate(
-                "key",
-                "group",
-                "id",
-                StreamGroupOptions.builder().makeStream().entriesRead("entry").build());
+                "key", "group", "id", StreamGroupOptions.builder().makeStream().entriesRead(123L).build());
         results.add(
                 Pair.of(
                         XGroupCreate,
                         buildArgs(
-                                "key", "group", "id", MAKE_STREAM_VALKEY_API, ENTRIES_READ_VALKEY_API, "entry")));
+                                "key", "group", "id", MAKE_STREAM_VALKEY_API, ENTRIES_READ_VALKEY_API, "123")));
```

Comment:
```suggestion
                                "key", "group", "id", MAKE_STREAM_VALKEY_API, ENTRIES_READ_VALKEY_API, 123L)));
```

Reply:
We have here strings only - those are redis command args in redis format

Reply:
Oops.  Ignore me.